tests/qa_automation/xfstests_device.pm

```diff
@@ -0,0 +1,115 @@
+# Copyright © 2012-2016 SUSE LLC
```

Comment:
Me neither , I think only lawyers good in copyright :)  But in my understanding you don't need copyright 2012-2017 for something which exists since 2017. So for any file created in 2017 copyright should look like `Copyright © 2017 SUSE LLC`.